### PR TITLE
Feature/nedry

### PR DESCRIPTION
**Pull request by SeveralCircles**

### CODE REVIEW

**Comment by SeveralCircles** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandNedry.java  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5533430&bid=18139476)

File: src/com/severalcircles/colors/commands/CommandNedry.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
         target.getWorld().spawnEntity(player.getLocation(), EntityType.PRIMED_TNT);
```
Comment (SeveralCircles) [inline src/com/severalcircles/colors/commands/CommandNedry.java]: This code really could be cleaned up a bit. Maybe use a for loop?

File: src/com/severalcircles/colors/commands/CommandNedry.java [hunk 2]
```diff
@@ -28,7 +28,7 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
...
         target.getWorld().spawnEntity(player.getLocation(), EntityType.PRIMED_TNT);
         int chatspam = 0;
-        while (chatspam < 3000) {
+        while (chatspam < 69) {
```
Comment (SeveralCircles) [inline src/com/severalcircles/colors/commands/CommandNedry.java:33]: You might be able to simplify this with a for loop instead of a while loop.

Comment (BeegBrainMo) [reply src/com/severalcircles/colors/commands/CommandNedry.java:33]: Made it a loop

**Review by SeveralCircles — COMMENTED** [review]
This code could still be improved, but you are free to merge it into develop.

**Pull request merged**